src/gui/advancedsettings.cpp

```diff
@@ -89,6 +89,7 @@ enum AdvSettingsRows
     // libtorrent section
     LIBTORRENT_HEADER,
     ASYNC_IO_THREADS,
+    STOP_TRACKER_TIMEOUT,
```

Comment:
Now it seems this order is out of place (I really overlook it). The new option should reside below `ANNOUNCE_IP`.
Anyone who is interested is welcomed to submit a PR.

Reply:
@an0n666 ping